src/UserContainer.java

```diff
@@ -1,9 +1,20 @@
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.Map;
 
 public class UserContainer {
+
+    static ArrayList<User> infoUser = new ArrayList();
    private final static Map<String, String> users = new HashMap<String, String>();
 
+public void containerSave() {
```

Comment:
this method creates new (empty) user and saves it to list?
Why do you need this